dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -33,6 +34,54 @@
 		compatible = "mmio-sram";
 	};
 
+	clocks {
+		clk_hse: clk-hse {
+			#clock-cells = <0>;
+			compatible = "st,stm32-hse-clock";
+			status = "disabled";
+		};
+
+		clk_hsi: clk-hsi {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <DT_FREQ_M(16)>;
+			status = "disabled";
+		};
+
+		clk_msi: clk-msi {
+			#clock-cells = <0>;
+			compatible = "st,stm32-msi-clock";
+			msi-range = <6>; /* 4MHz (reset value) */
+			status = "disabled";
+		};
+
+		clk_lse: clk-lse {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <32768>;
+			status = "disabled";
+		};
+
+		clk_lsi: clk-lsi {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <DT_FREQ_K(32)>;
+			status = "disabled";
+		};
+
+		pll: pll {
+			#clock-cells = <0>;
+			compatible = "st,stm32l4-pll-clock";
+			div-m = <1>;
+			mul-n = <1>;
+			div-p = <7>;
+			div-q = <2>;
+			div-r = <2>;
+			/* clock-output-names = "pllclk", "pllsai3clk", "pll48m1clk"; */
```

Comment:
Why is this property commented out?

Reply:
We're not ready to support that yet.
I'll remove